lib/active_fedora/aggregation/base_extension.rb

```diff
@@ -3,6 +3,13 @@ module BaseExtension
     extend ActiveSupport::Concern
     include PersistLinks
 
+    # Queries the RDF graph to find all aggregations that include this object
+    def aggregated_by
+      proxy_uris = self.resource.query(predicate: RDF::Vocab::ORE.proxyFor, object: RDF::URI(self.uri)).subjects.to_a
```

Comment:
This is the line where it should be finding the proxies but isn't finding anything.